config/packages/doctrine.yaml

```diff
@@ -0,0 +1,24 @@
+parameters:
+    # Adds a fallback DATABASE_URL if the env var is not set.
+    # This allows you to run cache:warmup even if your
+    # environment variables are not available yet.
+    # You should not need to change this value.
+
+doctrine:
+    dbal:
+        connections:
+            default:
```

Comment:
This will be separated into two databases. One for the events and the other for read models.